src/tiled/tilecollisiondock.cpp

```diff
@@ -80,6 +81,13 @@ TileCollisionDock::TileCollisionDock(QWidget *parent)
     CreateObjectTool *polygonObjectsTool = new CreatePolygonObjectTool(this);
     CreateObjectTool *templatesTool = new CreateTemplateTool(this);
 
+    // Autodetection of tile extents
+    QIcon autoDetectMaskIcon(QLatin1String(":images/22x22/stock-tool-fuzzy-select-22.png"));
+    QAction *autoDetectMask = new QAction(this);
+    autoDetectMask->setText(tr("Detect Shape"));
```

Comment:
This icon and text really suggests a more advanced functionality than actually implemented. I think the icon should include a rectangle and the text should say "Autodetect Collision Box".